app/controllers/AdminController.scala

```diff
@@ -128,6 +128,22 @@ class AdminController @Inject() (implicit val env: Environment[User, SessionAuth
     }
   }
 
+  /**
+   * Get a list of all tags used for the admin page.
+   */
+  def getAllTags = UserAwareAction.async { implicit request =>
+    val labels = LabelTable.selectTagsOfLabels(List())
+
+    val properties: List[JsObject] = labels.map (label => {
```

Comment:
Really minor, but I think the typical move is to not have a space here: `map (` because it's a function call.